language_detector/main.py

```diff
@@ -1,8 +1,21 @@
 # -*- coding: utf-8 -*-
 
-"""This is the entry point of the program."""
-
 def detect_language(text, languages):
-    """Returns the detected language of given text."""
-    # implement your solution here
-    pass
+    
+    globalcount = 0
+    lang = ""
+    
+    for dictionary in languages:
+        
+        count = 0
+        for word in text.split():
+            
+            if word in dictionary['common_words']:
+                count += 1
+                    
+        if count > globalcount:
+            globalcount = count
+            lang = dictionary['name']
```

Comment:
I like your `max()` approach because you also have all other languages' data in your dictionary. It wasn't requested, but it's a very good thing.